main.js

```diff
@@ -39,33 +51,66 @@ const getNodes = async () => {
   return nodes
 }
 
-const measure = async node => {
-  const measurement = {
-    node,
+const ping = async node => {
+  const partialMeasurement = {
     alive: false,
-    timeout: false,
+    timeout: null,
     ttfbMs: null
   }
   const start = new Date()
   try {
     const res = await fetch(
       `${node.protocol}://${node.host}:${node.port}/`,
       {
-        signal: AbortSignal.timeout(10_000)
+        signal: AbortSignal.timeout(PING_TIMEOUT)
       }
     )
     if (!res.ok) {
       throw new Error()
     }
   } catch (err) {
     if (err.name === 'TimeoutError') {
```

Comment:
```suggestion
    if (err instanceof TimeoutError) {
```

Reply:
> ReferenceError: Can't find variable: TimeoutError

https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout_static

I believe this doesn't work (yet)